assets/js/sections.js

```diff
@@ -0,0 +1,25 @@
+const propSections = {
```

Comment:
- [ ] Your project is currently lacking the current date display. For setting the time you are supposed to use the luxon library as suggested in the requirements. Please fix this.

Reply:
Yes we forgot it thanks.